Bolts/iOS/BFWebViewAppLinkResolver.m

```diff
@@ -8,7 +8,7 @@
  *
  */
 
-#import <UIKit/UIKit.h>
+#import <WebKit/WebKit.h>
```

Comment:
@roremeol just a nitpick, but in the comment inside `BFWebViewAppLinkResolver.h`, it is still saying `UIWebView`, may be you can also update that comment to `WKWebView`?

Reply:
> @roremeol just a nitpick, but in the comment inside `BFWebViewAppLinkResolver.h`, it is still saying `UIWebView`, may be you can also update that comment to `WKWebView`?